src/tableutils.js

```diff
@@ -74,17 +74,22 @@ export default class TableUtils extends Plugin {
 	 * @param {module:engine/model/position~Position} position Position at which insert a table.
 	 * @param {Number} rows Number of rows to create.
 	 * @param {Number} columns Number of columns to create.
+	 * @returns {module:engine/model/element~Element} Created table element.
 	 */
 	createTable( position, rows, columns ) {
 		const model = this.editor.model;
 
+		let table;
+
 		model.change( writer => {
```

Comment:
You can return table directly from `model.change()`. It fwds its callback's return value.